jar-connector/src/test/java/com/sixsq/slipstream/persistence/VmTest.java

```diff
@@ -36,8 +36,10 @@
 
```

Comment:
I would rather split tests :
* those specific to Vm stay in VmTest (but moved to proper project : Persistence)
* those using Collector in a new CollectorTest (same place as current VmTest).